cmd/apply/apply.go

```diff
@@ -117,8 +117,7 @@ func run(opts *Options) error {
 			// sort.Sort(aff)
 		}
 
-		//start to simulate scheduling
-		fmt.Printf(utils.ColorCyan+"There are %d pods to be scheduled\n"+utils.ColorReset, len(sim.GetPodsToBeSimulated()))
+		fmt.Printf(utils.ColorCyan+"There are %d pods to be simulated, and %d of them will be scheduled\n"+utils.ColorReset, sim.GetPodsCount(), sim.GetPodsWithoutNodeNameCount())
```

Comment:
fmt.Printf(utils.ColorCyan+"%d pods to be simulated,  %d pods of which to be scheduled\n"+utils.ColorReset, sim.CountPods(), sim.CountPodsWithoutNodeName())

Reply:
done